test/integration/SetProtocol.spec.ts

```diff
@@ -134,35 +134,29 @@ describe('SetProtocol', async () => {
     }
 
     test('returns the correct minimum natural unit', async () => {
-      const componentInstancePromises = _.map(mockTokens, mockToken => {
-        return DetailedERC20Contract.at(mockToken.address, web3, {});
-      });
-      const componentInstances = await Promise.all(componentInstancePromises);
-      const decimalPromises = _.map(componentInstances, componentInstance => {
-        return componentInstance.decimals.callAsync();
+      const erc20Wrapper = new ERC20Wrapper(web3);
```

Comment:
Let's move this wrapper to the top-level scope so that other tests can utilize erc20Wrapper as well.